Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml

```diff
@@ -584,12 +595,237 @@ script:
     - contextPath: OctoxLabs.Permissions.results.app
       description: 'Permission app.'
       type: String
-
+  - name: octoxlabs-search-user-inventory
+    description: 'Search in your User Inventory.'
+    arguments:
+      - default:
+        description: 'Query.'
+        name: query
+      - default:
+        description: 'Fields.'
+        isArray: true
+        name: fields
+      - default:
+        description: 'Page.'
+        name: page
+      - default:
+        description: 'Size.'
+        name: size
+      - default:
+        description: 'Specific Discovery Id.'
+        name: discovery_id
+    outputs:
+      - contextPath: OctoxLabs.UserInventory.count
+        description: 'Total users count.'
+        type: Number
+      - contextPath: OctoxLabs.UserInventory.results
+        description: 'List<Dict> User information.'
+        type: Unknown
+  - name: octoxlabs-search-applications
+    description: 'Search in your Applications'
+    arguments:
+      - default:
+        description: 'Query.'
+        name: query
+      - default:
+        description: 'Fields.'
+        isArray: true
+        name: fields
+      - default:
```

Comment:
In the code it looks like there is a default size: 1. It is better to add this in the description